control-plane-operator/controllers/hostedcontrolplane/oauth/deployment.go

```diff
@@ -170,6 +216,25 @@ func oauthVolumeConfig() *corev1.Volume {
 	}
 }
 
+func oauthVolumeAuditConfig() *corev1.Volume {
```

Comment:
nit: 
oauthVolumeAuditConfig / oauthAuditWebhookConfigFileVolume / buildOauthAuditWebhookConfigFileVolume

can we use "volume" consistently for new an existing funcs either as a suffix at the end or as oauthVolume prefix?

Reply:
@enxebre Sure, I could submit a dedicated pr to optimize it later. Thanks!